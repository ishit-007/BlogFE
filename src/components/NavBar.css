header{
  background-color: black;
  color: white;
  flex:none;
}

.headings{
  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  padding: 1rem;
  margin-left: 10%;
  padding-top: 2rem;
}
nav ul{
  list-style: none;
  display: flex;
  justify-content: flex-end;
  padding-bottom: 1rem;
}
nav li{
  padding-right: 10rem;
}

@media screen and (max-width:600px){
  nav li{
      padding-right: 5rem;
  }
}