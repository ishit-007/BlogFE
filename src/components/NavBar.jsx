import React from 'react';
import './NavBar.css'
import './Main.css'

const NavBar = () => {
  return (
    <header>
      <div className="headings">
        <h3>The Artifact</h3>
        <h4>
          <em>Culture & Art Blog</em>
        </h4>
      </div>

      <nav>
        <ul>
          <li>Blog</li>
          <li>About</li>
          <li>Contact</li>
        </ul>
      </nav>
    </header>
  );
};

export default NavBar;
