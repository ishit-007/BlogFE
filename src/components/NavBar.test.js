import { render } from '@testing-library/react';
import NavBar from './NavBar';
import renderer from 'react-test-renderer';
describe('NavBar', () => {
    it('should render the NavBar component', async() => {
        const tree=renderer.create(<NavBar/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});