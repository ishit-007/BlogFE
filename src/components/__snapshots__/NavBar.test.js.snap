// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render the NavBar component 1`] = `
<header>
  <div
    className="headings"
  >
    <h3>
      The Artifact
    </h3>
    <h4>
      <em>
        Culture & Art Blog
      </em>
    </h4>
  </div>
  <nav>
    <ul>
      <li>
        Blog
      </li>
      <li>
        About
      </li>
      <li>
        Contact
      </li>
    </ul>
  </nav>
</header>
`;
